src/logging/errorCodes.ts

```diff
@@ -0,0 +1,5 @@
+export type ErrorCodes = (typeof ErrorCodes)[keyof typeof ErrorCodes];
+
+export const ErrorCodes = {
```

Comment:
I think this should be `ErrorMessages`. `ErrorCodes` sounds like they'd be something numerical like HTTP Error Codes (404, 429, etc).

Are you naming this `ErrorMessages` because that's something defined on the server side?